ngsild-payloads/applicationPod.jsonld

```diff
@@ -0,0 +1,89 @@
+{
+  "id": "urn:ngsi-ld:ApplicationPod:istio-ingressgateway-9c8b9b586-thl7x",
+  "type": "ApplicationPod",
+  "name": {
+    "type": "Property",
+    "value": "istio-ingressgateway-9c8b9b586-thl7x"
+  },
+  "temporalResolution": {
+    "type": "Property",
+    "value": "PT1H"
+  },
+  "cpuUtilization": [
+    {
+      "type": "Property",
+      "observedAt": "2024-05-24T08:00:00Z",
+      "value": 0.06907686666721323,
+      "unitCode": "CORE"
```

Comment:
as per the spec, `unitCode` "shall be encoded using the UNECE/CEFACT Common Codes for Units of Measurement" (see www.unece.org/fileadmin/DAM/cefact/recommendations/rec20/rec20_Rev9e_2014.xls for instance). For instance, I guess this one should be P1 (percentage).